content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/try-ideas-to-manage-feedback-and-marking.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Try ideas to manage feedback and marking workload
+layout: /markdown_resource.html.erb
+---
+
+# Try ideas to manage feedback and marking workload
+
+<strong class="govuk-tag">Case study</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** King Charles I School
+
+**Location:** Worcestershire
+
+**Phase:** Secondary
+
+**Number of pupils:** 1000
+
+**Contact details:** Email Deputy Headteacher Ruth Allen at  <rallen@kingcharles1.worcs.sch.uk>
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        We made these changes to enable every teacher to become highly effective by:
+      </p>
+      <ul>
+        <li>
+          ensuring every teacher has time to focus on what is important - planning, teaching and feedback
+        </li>
+        <li>
+          believing in simplicity, always taking the shortest route and aiming for maximum impact on student learning with minimal workload for staff 
+        </li>
+        <li>
+          continuously reviewing and evaluating our systems in order to support all staff to achieve a healthy work life balance
+        </li>
+     </ul> 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ruth Allen, Deputy Headteacher
+
+{inset-text}
+
+At King Charles I school we know that teaching is exhilarating and rewarding but recognise that it can also be exhausting. We understand that time is precious and that tired teachers do not make effective teachers. Our aim is for staff to not have to take work home. 
```

Comment:
In the school details section it reads with a capital S - but I'm not sure which is correct! I think capital as it's name of school? Or can we not change it cos it's a quote?

```suggestion
At King Charles I School we know that teaching is exhilarating and rewarding but recognise that it can also be exhausting. We understand that time is precious and that tired teachers do not make effective teachers. Our aim is for staff to not have to take work home. 
```

Reply:
i think let's change for consistency like you suggest  :)